plugins/importexport/native/filter/NativeXmlPublicationFilter.inc.php

```diff
@@ -211,11 +211,12 @@ function parseIssueIdentification($publication, $node) {
 		}
 		$issueDao = DAORegistry::getDAO('IssueDAO'); /** @var $issueDao IssueDAO */
 		$issue = null;
-		$issuesByIdentification = $issueDao->getIssuesByIdentification($context->getId(), $vol, $num, $year, $titles);
-		if ($issuesByIdentification->getCount() != 1) {
+		$issuesByIdentification = $issueDao->getIssuesByIdentification($context->getId(), $vol, $num, $year, $titles)->toAssociativeArray();
+
+		if (count($issuesByIdentification) != 1) {
 			$deployment->addError(ASSOC_TYPE_PUBLICATION, $publication->getId(), __('plugins.importexport.native.import.error.issueIdentificationMatch', array('issueIdentification' => $node->ownerDocument->saveXML($node))));
 		} else {
-			$issue = $issuesByIdentification->next();
+			$issue = $issuesByIdentification[0];
```

Comment:
Since you're using `IssueDAO::toAssociativeArray`, I think `$issuesByIdentification[0]` would be looking for an issue ID of `0`, which won't exist.

Reply:
Fixed.

Reply:
Thanks! But I don't see the commit to change this pushed up -- did you forget to push your repo?

Reply:
I changed the toAssociativeArray() to toArray() two lines above.